### DISCUSSION TITLE
Firmware V5 U2AC

### DISCUSSION THREAD

**Discussion by ShadoViX** [ludwig-v/wireless-carplay-dongle-reverse-engineering#179] [General]
Please add to the firmware file list:
[U2AC_AUTOKIT_Update_20230712](https://github.com/ShadoViX/ShadoVsoftix)

**Comment by ludwig-v** [comment]
Hi ! Can you provide the full firmware version displayed on 192.168.50.2 ? 2023.07.12.XXXX ?

**Reply by azhiltsov** [reply]
I used this firmware to roll back from the recent 2024 update.
It works.
the full version string is 2023.07.12.1754

**Comment by sig1985** [comment]
Hi ! Can you provide the firmware version 2023.04.19 pls?
